hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -1372,6 +1372,10 @@ public boolean getGlobalSimpleIndexUpdatePartitionPath() {
     return getBoolean(HoodieIndexConfig.SIMPLE_INDEX_UPDATE_PARTITION_PATH_ENABLE);
   }
 
+  public int getNumBuckets() {
```

Comment:
lets make this more descriptive. `getBucketIndexNumBuckets()`?

Reply:
done